tools/server/src/settings.py

```diff
@@ -0,0 +1,190 @@
+from django.utils.translation import ugettext_lazy as _
```

Comment:
Sorry, I wasn't clear about what I'm commenting on. This file is full of comments and documentation that don't make any sense, like what to do in production. It would be best to clean up now, when we know what we're doing.